README.md

```diff
@@ -194,6 +194,42 @@ electricflow=cloudbees-flow-%s
 electricflow-minimumVersion=1.1.8
 ```
 
+### Running as on-demand GitHub Action
+
+```
+name: PCT Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      jenkins_ver:
+        description: 'Jenkins version'
+        required: true
+        default: '2.249.2'
+      branch:
+        description: 'Branch to test'
+        required: true
+        default: 'master'
```

Comment:
Would it not be more useful to do this on push/PR events, with the actual commit? So you can see if a _proposed_ change would break PCT, before merging?